### ISSUE TITLE
Inline frame missing a text alternative (comment form recaptcha)

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5417]
Sites identified: IJCCR
Page(s) identified: comments form

WCAG compliance level: A

Description:

Every inline frame (iframe) should have a text alternative that summarizes its content or purpose.

Inline frames (iframes) are used to insert content from other parts of the web.

A title that summarizes the visual content will help screen reader users to understand the iframe's purpose on the page.

Note: Archivo and MCO have been updated with a new captcha version that doesn't seem to have the recaptcha component. Could this be pushed to IJCCR easily?

See more details at [SiteImprove example](https://my2.siteimprove.com/Inspector/1167086/A11Y/Page?pageId=37972250838&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r13/failed/QW5JeW9GeUI=;M2QyYjIwMGM=).

**Comment by joncameron** [comment]
This might be on the gem we're using: https://github.com/avalonmediasystem/avalon/blob/d37bc1703085eef26affe53d9f23695148666179/app/views/comments/index.html.erb#L34.

For IU purposes we could update IJCCR with the v3 captcha, which should be an easy update.

**Comment by joncameron** [comment]
Closing; local issue for updating IJCCR instance: https://github.iu.edu/AvalonIU/Avalon/issues/181
